regparser/notice/build.py

```diff
@@ -114,6 +116,7 @@ def _check_local_version_list(url):
     notice_dir_suffix, file_name = os.path.split(path)
     for xml_path in settings.LOCAL_XML_PATHS:
         if os.path.isfile(xml_path + path):
+            print 'using ', xml_path + path
```

Comment:
So, this is another place where we should using `logging`, but I've already done so in #284. Obviously you've refactored this for the notice parsing, so it's in a different place.